### ISSUE TITLE
Dictionary without 'params.global' scope

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#251]
The following JSON code defines dictionaries in four different ways:

```
"testdict1" : {"subKey1" : {"subKey2" : {"subKey3" : {"subKey4" : 0}}}},
//
${testdict2.subKey1.subKey2.subKey3.subKey4} : 5,
//
${params.global.testdict3} : {"subKey1" : {"subKey2" : {"subKey3" : {"subKey4" : 10}}}},
//
${params.global.testdict4.subKey1.subKey2} : {"subKey3" : {"subKey4" : 20}}
```

In all cases `subKey4` has the correct value.

What astonishes me is that `testdict2` is created, even in case of inside the dollar operator brackets the scope '`params.global`' is not set.

I have no idea how to explain to a user, why the definitions of `testdict2` and `testdict3` are equivalent.

Shall the scope be required in such cases or not?

**Comment by test-fullautomation** [comment]
Hi Holger,
the 2. line is not correct.
It would create an element on root level, but this we don't do when we are in another scope.
$-syntax requires the full path to the current node.
Thank you,
 Thomas

**Comment by namsonx** [comment]
We now agreed that ```${testdict2.subKey1.subKey2.subKey3.subKey4} : 5,``` is invalid

**Comment by namsonx** [comment]
Hello Holger,

This enhancement is implemented in JsonPreprocessor package, I pushed the change to JP's stabi branch.

Thank you,
Son

**Comment by HolQue** [comment]
Retest successful. Issue can be closed.

**Comment by test-fullautomation** [comment]
Released with RobotFramework AIO 0.12.0
